msfr/steady/msfr_neutronics.i

```diff
@@ -6,9 +6,10 @@
 ################################################################################
 
 [Mesh]
+  uniform_refine = 1
```

Comment:
do you need that for neutronics? did you add this for TH?

Reply:
I added the same to TH yes. I mainly did this to avoid projections, but could work without it for neutronics.